src/jumpbox/setup

```diff
@@ -14,6 +14,6 @@ fi
 
 # install ruby
 
-rvm install ruby-2.3
-rvm --default use 2.3
-gem install --no-ri --no-rdoc bosh_cli bosh-workspace cf-uaac
+rvm install ruby-2.4
+rvm --default use 2.4
+gem install --no-ri --no-rdoc cf-uaac
```

Comment:
Because system may already exist, we should detect if bosh_cli gem is installed and remove it if it does.

Reply:
Good idea, thank you ! will do